File: JavaScript/Math-Array-String/03.js
// 2469. Convert the Temperature


/**
 * @param {number} celsius
 * @return {number[]}
 */

// var convertTemperature = function(celsius) {
//     Kelvin = celsius + 273.15
//     Fahrenheit = celsius * 1.80 + 32.00 
//     return [Kelvin, Fahrenheit]
// };

var convertTemperature = function(celsius) {
    return [celsius + 273.15 , celsius * 1.80 + 32.00]
};


console.log(convertTemperature(36.50)); // Output: [309.65, 97.7]

File: Python/Math-Array-String/03.py
# 2469. Convert the Temperature

from typing import List

class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
        return [celsius + 273.15 , celsius * 1.80 + 32.00]
    

celsius = 36.50

sol = Solution()
result = sol.convertTemperature(celsius)

print(result) # [309.65, 97.7]